src/main/java/com/google/sps/servlets/getArticlesServlet.java

```diff
@@ -184,4 +179,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("application/json");
     response.getWriter().println(json);
   }
+
+  public static void deleteResultsOfQueryFromDatastore(Query query, DatastoreService datastore) {
```

Comment:
The only difference for this method in three servlets is the type (article/game/video). Could you make it shared between all three servlets? You can pass in the type as a parameter.

Make sure you pass in the type as an enum if you have one, or create a global variable for each servlet. Do not use a local string.